Add minutes (#73)
sardana-org/sardana-followup: 20210120/MINUTES.md

            - adding `Start` method is a reasonable option
	      No updates
    - From the previous meetings:
        - [ ] SOLARIS: Cannot stop multiple motors movement in backlash direction [#1421](https://github.com/sardana-org/sardana/issues/1421). Status update
	      Update from Solaris: the problem was that if one moves one motor in backlash direction and make ctrl-C the motor stops
	      but if there are multiple motors the backlash is done and the people think that ctrl-C does not work.
	      They have investigated this and figured out that with one motor the flag for stopping is applied but in a group the flag is
	      applied to the group and not to all the elements. A PR is done where the stop flag is passed to each element of
	      the motor group. One has to decide the correct behaviour: Solaris suggest that a first ctrl-c would do the backlash
	      but if a second one is executed the backlash would not be done (stop/abort difference).
	      Daniel agrees with that and thinks it is the way the hardware does.
	      Zibi suggests to accept the current PR but create an new issue for distinguishing between stop and abort
	      (one/two ctrl-C)

> 	      A discussion about the motor group devices arised, the possibility of avoiding the creation will be analysed by
> 	      Zibi, it could be that they are not actually needed.

Reviewer: ```suggestion
	      A discussion about the motor group devices arised, the possibility of avoiding the creation of the Tango devices was proposed by @tiagocoutinho (in the past it was already mentione in [sardana-org/sardana#1338](https://github.com/sardana-org/sardana/issues/1338#issuecomment-637646445).
```

	   Not updates.
        - [ ] MAX IV will provide a docker image to add to ContinousIntegration.
	   It will be reported in the next meeting.
        - [ ] Shutter integration in continuos scans: status update.
	   No updates.
        - [ ] MaxIV beats configuration status: update from  Alejandro's work.
	   No news.
        - [ ] Add information on how to access debug information of libtango9-dbgsym (seems like some information is not accessible).
	   No updates.
        - [ ] Users can break Sardana by messing with configuration in Tango DB, diagnostic script should be created to check that.
	   No updates.
        - [ ] Configure CI on buster (problems with test suite on Python 3.6): status update.
	   Zibi will dedicate time to this. It could be an issue with the packages
	   coming from the distribution. The problems with the test suite are solved updating
	   tango. The CI part is related to github vs gitlab.

5. Overview of current developments / PR distribution
    - Problems for axis attributes with same name but different properties [#1436](https://github.com/sardana-org/sardana/issues/1436)

>       Not updates. Henry will do the programming.

Reviewer: ```suggestion
      Not updates (due to the holidays period) Henrik and Zibi will do the pair-programming.
```

      A PR is done, Carlos will review it.
    - Decouple attributes default values from memorized values [#1458](https://github.com/sardana-org/sardana/issues/1458)
      Not updates
    - SEP19
      Zibi will start soon. He will develop the unit test for the macromanager. Unit test will avoid regressions.
    - Step scans burst/points_per_step parameter [#1450](https://github.com/sardana-org/sardana/issues/1450)
      Related to the point "How to make scans depending on the status of the beam" (previous discussion)
    - pre- and post-move hooks not available in mv macro [#1471](https://github.com/sardana-org/sardana/issues/1471)
      A propostion: add pre-mv and post-mv to mv, have sardana custom setting to call them in a mv or not.
      Meanwhile work on general hooks with more granularity, add hooks to macros or groups of macros.
    - Access motor name of a mv or scans.
      It is proposed to add a motor attribute to all macros with pre/post-moves
    - PseudoCounters accessing pseudocounters.
      It does not work if sardana is reinitialized. It sounds like a bug.

>       Zibi will look into that.

Reviewer: ```suggestion
      @dschick will try to reproduce the problem and report it as an issue. 
```

    
6. Migration to gitlab
    Travis will not support the public OS for free, we are alredy using the budget
    of minutes. Tango will move to gitlab.
    The migration to gitlab will also imply a migration of the documentation, gitlab pages could host it.
    We could wait and see the experience of other projects.
    Solaris and MaxIV have their own repos in gitlab, but they agree on waiting.

>     We agree on keeping github.

Reviewer: ```suggestion
    We agree on keeping github for the moment at the same time not starting to use more github features (in order to not make the eventual future migration harder).
```

    
    
7. Sardana release Jan21
   - Who are the release managers?
     Zibi will check it.
   - Preselect milestone scope

   Solaris uses releases, the MBI will use them, so we decide that a release will be done.
   The milestones should be solved.
   https://github.com/sardana-org/sardana/milestone/8
   New issues could be added.

   The manual tests will be done in:

>    debian9, debian10, centos, windows, singularity (conda images - debian)

Reviewer: ```suggestion
   debian9, debian10, centos, windows, singularity (wrappling a docker image build on top of the conda base image)
```
